techeria_controller/techeria_db.sql

```diff
@@ -335,7 +335,56 @@ INSERT INTO `camera` (`id`, `image`, `name`, `price`, `description`, `category`)
 --
 -- Indexes for dumped tables
 --
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `accessorie`
+--
+
+CREATE TABLE `accessorie` (
+  `id` bigint(20) NOT NULL,
+  `image` varchar(100) NOT NULL,
+  `name` varchar(255) NOT NULL,
```

Comment:
make id as integer and auto increament. otherwise that will not work when seller want to upload their product